bramkouwen/Neatly-In-Motion
# Creating Contact Page

## What does this PR do?
* It creates contact.html 
* It creates the page for contact.html

## How do you check this PR?
* Pull down the branch
* Check the added contact.html file  
* Check the contact page

## Screenshots 
None.
